cmd/k8s-operator/sts.go

```diff
@@ -770,10 +820,13 @@ type ptrObject[T any] interface {
 // thing that changed is operator version (the hash is also exposed to users via
 // an annotation and might be confusing if it changes without the config having
 // changed).
-func hashBytes(b []byte) (string, error) {
-	h := sha256.New()
-	_, err := h.Write(b)
+func tailscaledConfigHash(c tailscaleConfigs) (string, error) {
+	b, err := json.Marshal(c)
```

Comment:
You can just do deephash.Hash(c).String()

Reply:
Discussed on Slack, comment above the function as to why we don't use our internal hash here